tests/ui-api-layer-acceptance-tests/internal/dex/dex.go

```diff
@@ -21,6 +21,10 @@ func SkipTestIfShould(t *testing.T) {
 }
 
 func SkipMainIfShould() {
+	if !isSCIEnabled() {
```

Comment:
In my opinion such method should have name like: ExitIfSCIEnabled or something like this. In the place where it is used nobody knows what it does.

Reply:
Right. I will rename it